CRM/Financial/BAO/FinancialTypeAccount.php

```diff
@@ -154,17 +154,19 @@ public static function del($financialTypeAccountId, $accountId = NULL) {
    * @return null|int
    */
   public static function getInstrumentFinancialAccount($paymentInstrumentValue) {
-    $paymentInstrument = civicrm_api3('OptionValue', 'getsingle', array(
-      'return' => array("id"),
-      'value' => $paymentInstrumentValue,
-      'option_group_id' => "payment_instrument",
-    ));
-    $financialAccountId = CRM_Contribute_PseudoConstant::getRelationalFinancialAccount(
-      $paymentInstrument['id'],
-      NULL,
-      'civicrm_option_value'
-    );
-    return $financialAccountId;
+    if (!isset(\Civi::$statics[__CLASS__]['instrument_financial_types'][$paymentInstrumentValue])) {
```

Comment:
Shouldn't this be `'instrument_financial_accounts'` ?

Reply:
@mattwire good spotting - fixed - is it good to merge now?